pom.xml

```diff
@@ -23,22 +23,32 @@
 
     <profiles>
         <profile>
-            <id>sum</id>
+            <id>sumAndSub</id>
             <activation>
                 <activeByDefault>true</activeByDefault>
             </activation>
             <properties>
-                <suite.file>src/test/resources/TestSuite1.xml</suite.file>
+                <suite.file>src\test\resources\SuiteTestSumAndSubtract.xml</suite.file>
             </properties>
         </profile>
 
         <profile>
-            <id>mult</id>
+            <id>MultAndDiv</id>
             <activation>
                 <activeByDefault>false</activeByDefault>
             </activation>
             <properties>
-                <suite.file>src/test/resources/TestSuite2.xml</suite.file>
+                <suite.file>src\test\resources\SuiteTestMultAndDiv.xml</suite.file>
+            </properties>
+        </profile>
+
+        <profile>
+            <id>AllTest</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+            </activation>
+            <properties>
+                <suite.file>src\test\resources\AllTestsSuite.xml</suite.file>
```

Comment:
Please do not use windows specific file/folder separators. This is does not work on the other OS. Also it could be a problem for the including tests in the CI process

Reply:
fix